### PR TITLE
Fix browserify and webpack bundling problems.

### PR DESCRIPTION
**Pull request by visortelle**
I had a problem with using this library as a dependency of other packages.
This fix solve the problem and all tests are passed succesfully.

### CODE REVIEW

File: browser-raw.js [hunk 1]
```diff
@@ -76,9 +76,9 @@ function flush() {
 
 // Safari 6 and 6.1 for desktop, iPad, and iPhone are the only browsers that
 // have WebKitMutationObserver but not un-prefixed MutationObserver.
-// Must use `global` instead of `window` to work in both frames and web
-// workers. `global` is a provision of Browserify, Mr, Mrs, or Mop.
-var BrowserMutationObserver = global.MutationObserver || global.WebKitMutationObserver;
+var BrowserMutationObserver = global ?
```
Comment (kriskowal) [inline browser-raw.js]: To function properly in workers and windows, we probably can use use `self`. Needs to be tested in both environments.

Comment (visortelle) [inline browser-raw.js]: There is no `self` in node environment. 

```
> npm run lint && npm run test-node


> asap@2.0.3 lint /Users/visortelle/vcs/asap
> jshint raw.js asap.js browser-raw.js browser-asap.js $(find scripts -name '*.js' | grep -v gauntlet)

browser-raw.js: line 79, col 31, 'self' is not defined.
browser-raw.js: line 79, col 56, 'self' is not defined.
```

Comment (visortelle) [inline browser-raw.js]: What's about:
```
var global = window || self || global;
var BrowserMutationObserver = global.MutationObserver || global.WebKitMutationObserver;
```

??

**Comment by kriskowal** [comment]
You will need to use the ternary operator with typeof for these not to be
reference errors in the environments in which they are not defined. It can
be done with just global and self.

**Comment by visortelle** [comment]
Do you mean something like this?
```
var BrowserMutationObserver = typeof global === 'object' ?
    (global.MutationObserver || global.WebKitMutationObserver) :
    (self.MutationObserver || self.WebKitMutationObserver);
```

**Comment by kriskowal** [comment]
Yes, or

```js
var scope = typeof global !== 'undefined' ? global : self;
var BrowserMutationObserver = scope.MutationObserver || scope.WebKitMutationObserver;
```

Note also that vars are hoisted so using the same variable on the left and right hand side would have odd behavior:

```js
var global = typeof global !== 'undefined' ? global : self;
```

In this case, it will always evaluate self because global exists in scope and typeof global is 'undefined'.

**Comment by visortelle** [comment]
Continue here [https://github.com/kriskowal/asap/pull/68](https://github.com/kriskowal/asap/pull/68)

**Pull request closed without merging**